lib/house.rb

```diff
@@ -0,0 +1,16 @@
+class House
+attr_reader :price, :address, :rooms
+  def initialize(price, address, rooms)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
```

Comment:
This is exactly the implementation that we want here, but without a test it is sad and lonely implementation.